phpcs.xml.dist

```diff
@@ -10,3 +10,13 @@
         <exclude name="PSR1.Methods.CamelCapsMethodName" />
     </rule>
 </ruleset>
+<phpunit bootstrap="vendor/silverstripe/framework/tests/bootstrap.php" colors="true">
+    <testsuites>
+        <testsuite name="sqlite3">
+            <directory>tests</directory>
+        </testsuite>
+        <testsuite name="framework">
+            <directory>vendor/silverstripe/framework/tests/php</directory>
+        </testsuite>
+    </testsuites>
+</phpunit>
```

Comment:
Add a newline at the the end of the file

Reply:
DONE